docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md

```diff
@@ -0,0 +1,97 @@
+---
+type: docs
+title: "How-To: Migrate from existing Kubernetes resources"
+linkTitle: "Migrate from YAML"
+description: "Learn how to migrate to a Radius container from existing Kubernetes YAML"
+weight: 500
+categories: "How-To"
+tags: ["containers","Kubernetes"]
+---
+
+This guide will provide an overview of how to migrate your existing Kubernetes resources from [Kubernetes YAML](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec).
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+### Step 1: Define a Radius application and container
+
+Define a Radius Application resource that will contain your Kubernetes resource container.
+
+> Radius resources, when deployed to a Kubernetes environment, are mapped to one or more Kubernetes objects. For more information, visit [Radius on Kubernetes platform]({{< ref "guides/operations/kubernetes#resource-mapping" >}}).
+
+{{< rad file="snippets/basemanifest.bicep" embed=true marker="//APPLICATION" >}}
```

Comment:
Let's remove this step entirely. Later on when we run `rad depploy` we can specify `-a demo` to create the application on the fly.